imager/adc-assembly/publish-model.conf

```diff
@@ -107,7 +107,7 @@ Current separation change rate of ADC calculated based on the rotation speed of
 Target orientation in FCRS<sub>IRIS-ROT</sub> of ADC at the time when this telemetry is issued.
 This target value is inferred from the following information:
 
-* orientation of elevation axis in FCRS<sub>IRIS</sub> (tcs.cm.iris.imgCurAtmDispersion eventstream)
+* orientation of elevation axis in FCRS<sub>IRIS-ROT</sub> (tcs.cm.iris.imgCurAtmDispersion eventstream)
```

Comment:
This is the correction of a mistake -- a number of months ago we renamed some coordinate systems, and it was always intended that the TCS provide us demands for most of our mechanisms in the rotating instrument coordinate system rather than the fixed (WRT NFIRAOS) coordinate system.

Based on this information, perhaps you can make any changes you feel are necessary directly in this branch (tcs_iris_09)? If/when you push to the copy of this branch in GitHub it should just appear as part of this same pull request.